utils/runtime-fuzzer/fuzz/fuzz_targets/main.rs

```diff
@@ -19,53 +19,13 @@
 #![no_main]
 
 use libfuzzer_sys::fuzz_target;
-use once_cell::sync::OnceCell;
-use std::{
-    fs::{self, OpenOptions},
-    io::{Error as IoError, Result as IoResult, Write},
-    path::Path,
-};
+use runtime_fuzzer::GearCalls;
 
-const SEEDS_STORE_DIR: &str = "/tmp/fuzzing-seeds-dir";
-const SEEDS_STORE_FILE: &str = "fuzzing-seeds";
-
-static RUN_INTIALIZED: OnceCell<String> = OnceCell::new();
-
-fuzz_target!(|seed: u64| {
+fuzz_target!(|gear_calls: GearCalls| {
     gear_utils::init_default_logger();
 
-    dump_seed(seed).unwrap_or_else(|e| unreachable!("internal error: failed dumping seed: {e}"));
-
-    log::info!("Running the seed {seed}");
-    runtime_fuzzer::run(seed);
+    // Newline to easily browse logs.
+    println!();
+    log::info!("Executing generated gear calls");
```

Comment:
You mix `println` (stdout) and `log::info` (may be filtered and redirected). Is it good? Maybe just add `\n` before the text in `log::info`?

Reply:
this code snipped can also be found at the other places in this PR

Reply:
discussed